Apps/AdminWebClient/src/ClientApp/package.json

```diff
@@ -17,6 +17,7 @@
     "vue": "^2.6.10",
     "vue-class-component": "^7.1.0",
     "vue-meta": "^2.3.2",
+    "vue-moment": "^4.1.0",
```

Comment:
Q: do you know what its the benefit of this one over the bare-bones moment library?

Reply:
A: my bad, I have missed your question on rocket chat last time. I didn't know that we were using momentJS on the WebClient project. It is the same. I will remove the vue-moment from the WebAdmin for now, we will add the momentJS to WebAdmin when we really need it.